go.sum

```diff
@@ -1,3 +1,5 @@
+connectrpc.com/connect v1.11.1 h1:dqRwblixqkVh+OFBOOL1yIf1jS/yP0MSJLijRj29bFg=
+connectrpc.com/connect v1.11.1/go.mod h1:3AGaO6RRGMx5IKFfqbe3hvK1NqLosFNP2BxDYTPmNPo=
```

Comment:
There are deps on github.com/bufbuild/connect-go below. Does this need tidying up, or does it really depend on both?